docs/source/markdown/podman-push.1.md

```diff
@@ -91,19 +91,19 @@ solely for scripting compatibility.
 #### **--format**, **-f**=*format*
 
 Manifest Type (oci, v2s1, or v2s2) to use when pushing an image to a directory using the 'dir:' transport (default is manifest type of source)
-Note: This flag can only be set when using the **dir** transport
+Note: This flag can only be set when using the **dir** transport. (Not available for remote commands)
```

Comment:
Suggest rewording all of these to "Not available with the remote client"

Reply:
Mheon I am going to merge this and then change the man pages as a docs only change.